# This contains an array of custom type Point2D
Point2D[] points
int64 length